framework/src/postprocessors/FindValueOnLine.C

```diff
@@ -28,6 +28,15 @@ FindValueOnLine::validParams()
   params.addParam<Point>("start_point", "Start point of the sampling line.");
   params.addParam<Point>("end_point", "End point of the sampling line.");
   params.addParam<Real>("target", "Target value to locate.");
+  params.addParam<bool>(
+      "stop_if_not_found",
+      true,
+      "If true, stop with error if target value is not found on the line. If false, "
+      "return not_found_val.");
+  params.addParam<Real>("not_found_val",
+                        -1,
+                        "Value to return if target value is not found on line and "
+                        "stop_if_not_found is false.");
```

Comment:
Would you be alright with changing the name of these parameters to "error_if_not_found" and "default_value"?

Reply:
Good idea, done